configuration/src/configuration.ts

```diff
@@ -45,8 +47,15 @@ export interface ConfigOptions {
     logger?: Function;
 }
 
+export interface Secret {

```

Comment:
This is a type that the config library owns and exposes to its consumers, so I don't think typings/ is a good place for it (afaik, it's more for the library's dependencies)